features/collections.feature

```diff
@@ -139,6 +139,28 @@ Feature: Collections
     And the _site directory should exist
     And the "_site/puppies/fido.html" file should not exist
 
+  Scenario: Hidden collection with document with future date, accessed via Liquid
```

Comment:
This is nitpicky, but could you change this to `Hidden collection has document with future date, accessed via Liquid`

I had trouble parsing the scenario description until I read the full test.

Reply:
Sure, no problem. I was just keeping with the phrasing on the prior test. That'll be fixed in just a sec.